codalab/worker/singularity_utils.py

```diff
@@ -0,0 +1,16 @@
+def is_shub(img):
+    return img.startswith("shub://")
+
+
+def is_sylabs(img):
+    return img.startswith("library://")
+
+
+def is_singularity(image_spec):
```

Comment:
This function isn't used

Reply:
I am planning on using it in the next PR for singularity support. That function will determine whether or not to run in a singularity runtime. I would like to keep it for now, and possibly remove it later.

Reply:
I think it's better to remove it and then re-add it in the future PR. I'm not able to give meaningful feedback for this function since I don't know how it's going to be used. If we want to keep it, though, we should add a docstring and explain how it can be used.